contracts/interchain-token-service/src/msg.rs

```diff
@@ -51,4 +54,6 @@ pub enum QueryMsg {
     /// Query all registererd ITS contract addresses
     #[returns(HashMap<ChainNameRaw, Address>)]
     AllItsContracts,
+    #[returns(Vec<(TokenId, nonempty::String)>)]
```

Comment:
Wanted to use HashMap here, but serde actually doesn't let you use a new type as a map key. The code panics

Reply:
Not sure I follow, why does it work for `AllItsContracts` instead?

Reply:
It works because that type is annotated so deserialization is from a string. I switched it back to HashMap by just swapping the types, so it will map denom to token id, which I think is better for a caller anyways (they probably know the denom but not the token id).